bin/deps.ml

```diff
@@ -0,0 +1,40 @@
+(*
+ * Copyright (c) 2018 Ulysse Gérard <ulysse@gtarides.com>
```

Comment:
```suggestion
 * Copyright (c) 2020 Ulysse Gérard <ulysse@gtarides.com>
```

Reply:
😅

Reply:
The same typo in `dep.ml`, `dep.mli`, `deps.mli`